### PR TITLE
Consolidate common snippet logic

### LINKED ISSUE #47: Consolidate language-independent snippet files
It should be possible to reference common, language independent snippet files that provide utilities, such as boolean operators. Right now, several language duplicate such code.

### PR DESCRIPTION
**Pull request by jmuk**
Fixes #47

### CODE REVIEW

**Comment by jmuk** [comment]
FYI @geigerj

File: vgen/src/main/resources/io/gapi/vgen/common/common.snip [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# TODO(jgeiger): move to non-Python-specific location.
```
Comment (geigerj) [inline vgen/src/main/resources/io/gapi/vgen/common/common.snip]: Can we remove the "common" directory for shared snippets, and just keep them one level higher on the directory tree?

Comment (jmuk) [inline vgen/src/main/resources/io/gapi/vgen/common/common.snip]: done

**Comment by geigerj** [comment]
LGTM

**Comment by garrettjonesgoogle** [comment]
LGTM

**Pull request merged**